verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -1312,9 +1312,19 @@ absl::string_view StripOuterQuotes(absl::string_view text) {
 
 void IndexingFactsTreeExtractor::ExtractInclude(
     const verible::SyntaxTreeNode& preprocessor_include) {
-  const SyntaxTreeLeaf& included_filename =
+  const verible::Symbol& included_filename_symbol =
       GetFileFromPreprocessorInclude(preprocessor_include);
 
+  // Terminate if this isn't a string literal.
+  if (included_filename_symbol.Kind() != verible::SymbolKind::kLeaf ||
```

Comment:
if you call `Tag()` once, that will also give you `.kind` (and avoid double virtual call).